test/webrick/test_httprequest.rb

```diff
@@ -312,6 +312,30 @@ def test_bad_chunked
     end
   end
 
+  def test_bad_chunked_extra_data
+    msg = <<-_end_of_message_
```

Comment:
Do you mind updating this test to use squigly heredocs?

Reply:
Sure thing, pushed a rebased commit for that.